Extensions.md

```diff
@@ -14,7 +14,7 @@
 -->
 # Apache MiNiFi Extensions Guide
 
-To enable all extensions for your platform, you may use -DENABLE_ALL=TRUE OR select option m in the bootstrap.sh guided build process defined in the [ReadMe](https://github.com/apache/nifi-minifi-cpp/#bootstrapping)
+To enable all extensions for your platform, you may use -DENABLE_ALL=TRUE OR select the option for "Enabling all Extensions" in the bootstrap script. [ReadMe](https://github.com/apache/nifi-minifi-cpp/#bootstrapping)
```

Comment:
"Enable all extensions"

Reply:
ah, this was (sadly) intentional as  "Enable all extensions" doesn't really do enable everything. It was meant to be "air quotes that imply something but not really..." 

I was planning on adding more to this at a later time, so I think I will clarify that not all extensions can be enabled for all platforms instead of using the odd wording I chose. Thanks!